tensorflow/compiler/mlir/tensorflow/tests/lower_tf.mlir

```diff
@@ -276,6 +276,13 @@ func @fourdim_space_to_batch_nd(%input: tensor<3x5x7x10xf32>, %block_shape: tens
   return %0 : tensor<?x?x?x10xf32>
 }
 
+// %Input has elements of type f16.
+// CHECK-LABEL: space_to_batch_nd_element_type_f16
+func @space_to_batch_nd_element_type_f16(%input: tensor<3x5x7x10xf16>, %block_shape: tensor<2xi64>, %paddings: tensor<2x2xi64>) -> tensor<?x?x?x10xf16> {
+  %0 = "tf.SpaceToBatchND"(%input, %block_shape, %paddings) : (tensor<3x5x7x10xf16>, tensor<2xi64>, tensor<2x2xi64>) -> tensor<?x?x?x10xf16>
+  return %0 : tensor<?x?x?x10xf16>
```

Comment:
Can you add CHECK lines in this function to actually test that it does what you expect?

Reply:
Added the check lines along with better comment for the test case. Thanks !